packages/graphql-elasticsearch-transformer/src/resources.ts

```diff
@@ -45,13 +45,6 @@ export class ResourceFactory {
                 Description: 'The number of instances to launch into the Elasticsearch domain.',
                 Default: 1
             }),
-            [ResourceConstants.PARAMETERS.ElasticsearchDomainName]: new StringParameter({
```

Comment:
Why are we removing this?

Reply:
This is no longer used and is not documented so I am removing it.